linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/tests/rhel7_correct_value.pass.sh

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_all
```

Comment:
I can see that it has `rhel7` in the file name so you aren't consistent here. Moreover, `multi_platform_all` is the default value so we don't put it into our test scenarios.